Gruntfile.js

```diff
@@ -36,25 +36,51 @@ module.exports = function (grunt) {
       },
       concat: {
         options: {
-          separator: ';'
+          separator: '\n'
```

Comment:
I think since we are post-processing/minifying our `js`, we want to keep this separator as a `;`, just in case.  I think its a guard against someone forgetting to end a file with a semi, though a `\n` may solve the same problem.

Reply:
Can we do something like ";\n"? When looking at the resulting minified files, at least one newline made it easier to see where one file ended and the next started. I was inspecting the results of the build and they were just a little difficult to read sometimes.

Reply:
Yup, that works for me.